### DISCUSSION TITLE
Thinner recursive datasets

### DISCUSSION THREAD

**Discussion by viharm** [psy0rz/zfs_autobackup#260] [Q&A]
Does the thinner act recursively on the target?

I was reviewing the debug log of some trial runs and noticed that zfs-autobackup enumerates the child datasets of the target and lists their snapshots.

Seems unnecessary. But is it thinning the target recursively? Including the siblings of the desired target dataset?

Am I missing something?

**Comment by psy0rz** [comment]
This was not clear in the documentation, so i've added it:

https://github.com/psy0rz/zfs_autobackup/wiki/Thinner#which-snapshots-is-it-operating-on

(answer is yes)
